runmactest

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+PATH_TO_RELEASE_DOT_APP=/Users/simondemeule/GitHub/build-autonomx-Desktop_Qt_5_15_0_clang_64bit-Release/autonomx/autonomx.app
```

Comment:
Can we avoid hard-coding the path to `/Users/simondemeule/GitHub`, so that other developers can also use this script ?